lab27/src/StarWarsActor.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment, useState } from 'react';
+import useSWR from 'swr';
+import StarWarsFilm from './StarWarsFilm';
+
+const StarWarsActor = () => {
+  const [actorId, setActorId] = useState(1);
+  const { data, error } = useSWR(`https://swapi.dev/api/people/${actorId}/`);
+  return (
+    <Fragment>
+      <input
+        type="text"
+        value={actorId}
+        onChange={e => setActorId(e.target.value)}
+      />
+      {data && (
+        <section className="star-wars">
+          <p> Name: {data.name}</p>
+          <p> height: {data.height}</p>
+          <p>
+            <u>Films:</u>
+          </p>
+          {data.films.map(film => (
```

Comment:
So we talked about this today - it's always a good idea to limit the number of results you show on screen (regardless of what the server returned)